test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -85,6 +85,10 @@ type clusterUpgradeWithRuntimeSDKSpecInput struct {
 	// Allows to inject a function to be run after test namespace is created.
 	// If not specified, this is a no-op.
 	PostNamespaceCreated func(managementClusterProxy framework.ClusterProxy, workloadClusterNamespace string)
+
+	// Allows to inject a function to be run after the cluster is upgraded.
```

Comment:
Given how we usually use provisioned I think we should use a func name which matches the godoc more, e.g. what about "PostUpgrade"?

Reply:
Renamed to PostUpgrade 👍